functions

```diff
@@ -796,10 +796,14 @@ install_modules() {
             *.gz)
                 gz_comp+=("$moduledest/kernel/${m##*/}")
                 ;;
+            *.zst)
+                zst_comp+=("$moduledest/kernel/${m##*/}")
+                ;;
         esac
     done
     (( ${#xz_comp[*]} )) && xz -d "${xz_comp[@]}"
     (( ${#gz_comp[*]} )) && gzip -d "${gz_comp[@]}"
+    (( ${#zst_comp[*]} )) && zstd -d "${zst_comp[@]}"
```

Comment:
I believe `zstd` should be silenced here with the `-q` option in order to avoid garbage during initramfs regeneration both in console and pacman log.

Reply:
You're absolutely right, I overlooked that.